test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SigningTestFixture.cs

```diff
@@ -201,7 +202,7 @@ private async Task<CertificateAuthority> CreateDefaultTrustedCertificateAuthorit
             _trustedServerRoot = TrustedTestCert.Create(
                 rootCertificate,
                 StoreName.Root,
-                StoreLocation.LocalMachine);
+                (RuntimeEnvironmentHelper.IsWindows || RuntimeEnvironmentHelper.IsMacOSX)? StoreLocation.LocalMachine : StoreLocation.CurrentUser);
```

Comment:
see above comment about store locations.

Reply:
The background is https://github.com/dotnet/runtime/blob/master/docs/design/features/cross-platform-cryptography.md#x509store, added in the comments. Thanks!